docs/UserGuide.md

```diff
@@ -379,7 +384,8 @@ Format: `markstudent li/LESSON_INDEX si/STUDENT_INDEX`
 * `LESSON_INDEX` and `STUDENT_INDEX` **must be a positive integer** 1, 2, 3, …​
 
 Example:
-* `markstudent li/2 si/3` marks the 3rd student in the 2nd lesson as present.
+* `markstudent li/2 si/3` marks the 3rd student in the student list within the 2nd lesson in your lesson list as present.
```

Comment:
I think this phrasing is unnecessary

The phrasing before is clear enough imo since the paragraph above already explains what STUDENT_INDEX refers to

Reply:
I think I will remove the `in the student list part within` part, but keep the back part for `in your lesson list` to be more consistent with previous example